release-please-config.json

```diff
@@ -9,6 +9,10 @@
     },
     "libraries-bom": {
       "component": "libraries-bom"
+    },
+    "tests": {
+      "component": "tests",
```

Comment:
This is my attempt. I hope this tells Release Please to manage the `{x-version-update:google-cloud-bom:current}` mark in tests directory.

Reply:
@suztomo This might work because release-please will not be able to find a component "tests" which should (ideally) be a releasable entity under the "tests" package. Generally "component" is the main artifact which is to be released for that package. Remember that for google-cloud-java dummy release pr, it contained [component](https://github.com/googleapis/google-cloud-java/pull/7992#:~:text=google%2Dcloud%2Daccessapproval%3A) name and not package name.